services/api/config/application.yml.example

```diff
@@ -18,25 +18,35 @@ production:
   secret_token: ~
   blob_signing_key: ~
   sso_app_secret: ~
-  sso_app_id: ~
-  sso_provider_url: ~
-  workbench_address: ~
-  websocket_address: ~
+  sso_app_id: arvados-server
+  sso_provider_url: https://uuid_prefix.your.domain
+  workbench_address: https://workbench.uuid_prefix.your.domain
+  websocket_address: wss://ws.uuid_prefix.your.domain/websocket
   #git_repositories_dir: ~
   #git_internal_dir: ~
+  #git_repo_ssh_base: "git@git.uuid_prefix.your.domain:"
+  #git_repo_http_base: https://git.uuid_prefix.your.domain/
+  #auto_admin_first_user: false
+  #crunch_job_wrapper: :slurm_immediate
+  #sso_insecure: false
```

Comment:
The lines I added was from the [Git](http://doc.arvados.org/install/install-arv-git-httpd.html) and [Crunch](http://doc.arvados.org/install/install-crunch-dispatch.html) install doc. They both said "add the line".
I arranged the order of options in the same way as ```application.default.yml```. I've reordered them on your request.